Game of Game of Life.py

```diff
@@ -0,0 +1,98 @@
+### a,b refers to cell (a,b) whereas x,y refers to pixel coordinates ###
+
+import pygame as pg, math as m, time as t, random as r
+from config import *
+
+class Cell():
+    def __init__(self):
+        self.CurrentState=1
+        self.NextState=0
+    def Kill(self):
+        self.NextState=0
+    def Birth(self,Type):
+        self.NextState=Type
+
+def Check(a,b):
+    """Checks whether the cell will be dead or alive at the end of this turn,
+    and if so what type it will be"""
+    Type=Board[a][b].CurrentState
+    total=[0,0]
+    total[Board[a+1][b].CurrentState]+=1
+    total[Board[a][b+1].CurrentState]+=1
+    total[Board[a-1][b].CurrentState]+=1
+    total[Board[a][b-1].CurrentState]+=1
+    total[Board[a+1][b+1].CurrentState]+=1
+    total[Board[a-1][b-1].CurrentState]+=1
+    total[Board[a+1][b-1].CurrentState]+=1
+    total[Board[a-1][b+1].CurrentState]+=1
+    Next=0
+    if Type==0:
+        if total[0]==5:# if 5 dead cells; ie. if 3 alive cells
+            Next=1
+    if Type==Square:
+        if total[0]==5 or total[0]==6:
+            Next=1
+    return Next
+
+def CleanBoard():
+    return [[Cell()for a in range(Height+(2*Cushion))]for b in range(Width+2*Cushion)]
+
+def Draw(Type,a,b,colour):
+    """Draws a type of cell (Type) at the desired cell (a,b)"""
+    x=(a-Cushion)*Size+Edge/2
+    y=(b-Cushion)*Size+Edge/2
+    s=Size-Edge
+    pg.draw.rect(Screen,(255,255,255),(x,y,s,s))
+
+    if Type==Square:
+        pg.draw.rect(Screen,colour,(x,y,s,s))
+
+def CheckUserInput():
+    global Paused
+    global OneTurn
```

Comment:
Remove globals and make this function return 2 variables. 
```
def func():
...
return Var1, Var2

...

Var1, Var2 = func()
```

Reply:
done